gdxpds/gdx.py

```diff
@@ -267,6 +267,74 @@ def read(self,filename):
                 symbol.load()
         return
 
+    def read_single_symbol(self,filename,target_symbol_name):
+        """
+        Optimized read method that only loads metadata for a specific symbol.
+
+        Parameters
+        ----------
+        filename : pathlib.Path or str
+            Path to the GDX file to read
+        target_symbol_name : str
+            Name of the specific symbol to read
+
+        Returns
+        -------
+        GdxSymbol
+            The requested symbol (not yet loaded with data)
+
+        Raises
+        ------
+        Error
+            If not self.empty or if symbol not found
+        """
+        if not self.empty:
+            raise Error("GdxFile.read_single_symbol can only be used if the GdxFile is .empty")
```

Comment:
Does this mean that you can't call this function multiple times to read in the metadata for multiple symbols? So you can either read in the meta data for all symbols or for just one symbol?